openhands/runtime/client/client.py

```diff
@@ -327,6 +333,7 @@ async def run_ipython(self, action: IPythonRunCellAction) -> Observation:
             obs: IPythonRunCellObservation = await _jupyter_plugin.run(action)
             obs.content = obs.content.rstrip()
             obs.content += f'\n[Jupyter current working directory: {self.pwd}]'
+            obs.content += f'\n[Jupyter Python interpreter: {_jupyter_plugin.python_interpreter_path}]'
```

Comment:
Is this really helping the llm and/or does it have to be in each and every observation?
_Token cost!_ 😄